autoconnect/autoconnect-web/src/dockerflow.rs

```diff
@@ -27,19 +28,33 @@ pub fn config(config: &mut web::ServiceConfig) {
 
 /// Handle the `/health` and `/__heartbeat__` routes
 pub async fn health_route(state: Data<AppState>) -> Json<serde_json::Value> {
-    let healthy = state
+    let mut health = json!({
```

Comment:
Can you add this to quiet warnings when the feature's disabled
```suggestion
    #[allow(unused_mut)]
    let mut health = json!({
```